csrankings.csv

```diff
@@ -15787,3 +15787,56 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
+Alan Burns,University of York,https://www-users.cs.york.ac.uk/~burns/,Io7KbYcAAAAJ
+Ibrahim Habli,University of York,https://www-users.cs.york.ac.uk/~ihabli/,U9j-FS8AAAAJ
+Tangming Yuan,University of York,https://www-users.cs.york.ac.uk/~tommy/,1fzy4fsAAAAJ
+Lilian Blot,University of York,https://www-users.cs.york.ac.uk/~lblot/,Enkx56UAAAAJ
+Radu Calinescu,University of York,https://www-users.cs.york.ac.uk/~raduc/,NLiNqPwAAAAJ
+Daniel Kudenko,University of York,https://www-users.cs.york.ac.uk/~kudenko/,uUXUMukAAAAJ
+Daniel W. Franks,University of York,https://danfranks.weebly.com,Zpo8koYAAAAJ
+Javier Cámara,University of York,http://www.javicamara.com/,iMMncw8AAAAJ
+Tim P. Kelly,University of York,https:/www-users.cs.york.ac.uk/~tpk/,cNeDb_8AAAAJ
+Dimitar Kazakov,University of York,https://www-users.cs.york.ac.uk/~kazakov/,CLI_4LYAAAAJ
+Stefano Pirandola,University of York,https://www-users.cs.york.ac.uk/~pirs/,bV8llfkAAAAJ
+Christopher Power,University of York,https://www-users.cs.york.ac.uk/~cpower/,1ZA0JtcAAAAJ
+Paul Cairns,University of York,https://www-users.cs.york.ac.uk/~pcairns/,cNVmuG0AAAAJ
+Helen Petrie,University of York,https://www-users.cs.york.ac.uk/~petrie/,xhkqoGAAAAAJ
+Suresh Manandhar,University of York,https://www-users.cs.york.ac.uk/~suresh/,5iH8GVIAAAAJ
+Simon O'Keefe,University of York,https://www-users.cs.york.ac.uk/~sok/,QL1JHlUAAAAJ
+Ana Cavalcanti,University of York,https://www-users.cs.york.ac.uk/~alcc/,fu6TM4wAAAAJ
+Siamak F. Shahandashti,University of York,https://www-users.cs.york.ac.uk/~siamak/,CZTajNQAAAAJ
+Susan Stepney,University of York,https://www-users.cs.york.ac.uk/~susan/,rD32w_EAAAAJ
+Edwin R. Hancock,University of York,https://www-users.cs.york.ac.uk/~erh/,EjDU2ncAAAAJ
+Adrian G. Bors,University of York,https://www-users.cs.york.ac.uk/~adrian/,cvdyalUAAAAJ
+Detlef Plump,University of York,https://www-users.cs.york.ac.uk/~det/,nh6AHbIAAAAJ
+Nicholas Drivalos Matragkas,University of York,https://www-users.cs.york.ac.uk/~nikos/,Dvp7fNoAAAAJ
+John A McDermid,University of York,https://www-users.cs.york.ac.uk/~jam/,REscLkoAAAAJ
+William A. P. Smith,University of York,https://www-users.cs.york.ac.uk/wsmith/,o2z0h6MAAAAJ
+Anders Drachen,University of York,https://www.cs.york.ac.uk/people/?group=Academic%20and%20Teaching%20Staff&username=adrachen,rOcL0NgAAAAJ
+Samuel L. Braunstein,University of York,https://www-users.cs.york.ac.uk/~schmuel/,jh-mlXoAAAAJ
+Angus M. Marshall,University of York,https://www-users.york.ac.uk/~amm616/,RV4F9n8AAAAJ
+Chris Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb/,QQ_dWtcAAAAJ
+Steven A. Wright,University of York,https://www-users.cs.york.ac.uk/~saw/,uh1UaD4AAAAJ
+Dimitrios S. Kolovos,University of York,https://www-users.cs.york.ac.uk/~dkolovos/,aRd1i6UAAAAJ
+James Cussens,University of York,https://www-users.cs.york.ac.uk/~jc/,OEFkOR0AAAAJ
+Steve King,University of York,https://www-users.cs.york.ac.uk/~king/,Q75-BIgAAAAJ

```

Comment:
Needs to be "Steve King 0001". See https://dblp.uni-trier.de/pers/hd/k/King_0001:Steve